utils/merge.py

```
@@ -1,91 +1,174 @@
+    dest_path = None
+    output_pattern = r"^\d+\.\s+(.+)$"
+    for line in result.stdout.splitlines():
+        match = re.match(output_pattern, line.strip())
+        if match:
+            dest_path = Path(match.group(1))
+            break
+
+    if dest_path:
+        book.dest_path = str(dest_path)
+        book.save(update_fields=["dest_path"])
+        logger.info(f"Output file: {dest_path}")
```

Ruff (0.15.0)

[warning] Logging statement uses f-string

(G004)